### ISSUE TITLE
Commissions are not available in `run_algorithm` output

### ISSUE THREAD

**Issue by AdrianTeng** [quantopian/zipline#2137]
Dear Zipline Maintainers,

Before I tell you about my issue, let me describe my environment:

# Environment

* Operating System: Linux ubuntu 4.13.0-37-generic #42~16.04.1-Ubuntu SMP Wed Mar 7 16:03:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
* Python Version: `3.5.5`
* Python Bitness: `64`
* How did you install Zipline: local development

Now that you know a little about me, let me tell you about the issue I am
having:

# Description of Issue

The output dataframe of `run_algorithm` contains a column called `transactions`, which contains all transactions of the strategy for each day. However, all transactions have `None` in their `commissions` attribute.

This could be fixed by adding line `txn.commission = additional

https://github.com/quantopian/zipline/commit/1f50b1b0d3b4f1a43b22a111196467b34b4c0971#diff-82f717facf7980359a913c2407bed230R62

* What did you expect to happen?
* What happened instead?

...

Sincerely,
`$ whoami`

**Comment by AdrianTeng** [comment]
Would @llllllllll be able to help me with this?

**Comment by imkoukou** [comment]
Hi, I think you can call zipline function of context.set_commission( ) in the initialize function to specify transaction commission.

**Comment by AdrianTeng** [comment]
@imkoukou Yes, setting commission works, but you don't know how much you pay in commission in total for your simulation (which is what this question is about)
